solc/CommandLineParser.cpp

```diff
@@ -328,33 +328,42 @@ bool CommandLineParser::parseInputPathsAndRemappings()
 	m_options.input.ignoreMissingFiles = (m_args.count(g_strIgnoreMissingFiles) > 0);
 
 	if (m_args.count(g_strInputFile))
-		for (string path: m_args[g_strInputFile].as<vector<string>>())
+		for (string positionalArg: m_args[g_strInputFile].as<vector<string>>())
 		{
-			auto eq = find(path.begin(), path.end(), '=');
-			if (eq != path.end())
+			if (ImportRemapper::isRemapping(positionalArg))
 			{
+				optional<ImportRemapper::Remapping> remapping = ImportRemapper::parseRemapping(positionalArg);
+				if (!remapping.has_value())
+				{
+					serr() << "Invalid remapping: \"" << positionalArg << "\"." << endl;
```

Comment:
I'm fine with either but the whole file is using `endl` right now so I'd rather keep it like that in this PR.

Maybe you could create a separate issue for this and mark it as `good first issue`? It's a simple refactor that could be done by an external contributor but here it would only add noise.